pkg/reconciler/labeler/labeler.go

```diff
@@ -49,32 +49,64 @@ var _ routereconciler.Interface = (*Reconciler)(nil)
 var _ routereconciler.Finalizer = (*Reconciler)(nil)
 
 func (c *Reconciler) FinalizeKind(ctx context.Context, r *v1.Route) pkgreconciler.Event {
-	switch cfgmap.FromContextOrDefaults(ctx).Features.ResponsiveRevisionGC {
+	newGC := cfgmap.FromContextOrDefaults(ctx).Features.ResponsiveRevisionGC
 
-	case cfgmap.Disabled: // v1 logic
+	// v1 logic
+	if newGC == cfgmap.Disabled || newGC == cfgmap.Allowed {
```

Comment:
So allowed labels  both ways?

Reply:
Correct! This allows you to test allowed without destroying your lastPinned data and later fully upgrade.